cli/src/main.rs

```diff
@@ -237,15 +237,30 @@ fn run() -> Result<(), CliError> {
                     .takes_value(true)
                     .help("Path to private key file"),
             )
+            .arg(
+                Arg::with_name("node_file")
+                    .long("node-file")
+                    .takes_value(true)
+                    .required_unless("node")
+                    .long_help(
+                        "File system path or URL to a .yaml file of node definitions. \
```

Comment:
This description is too long. Limit it to the first sentence. Remove the period at the end (these should all be the same). The rest could go in a details section below (there is an example in the generate cert command I believe).

Change .yaml to YAML or something like "Filesystem path or HTTP URL to nodes file", because the format needs more explanation than just "YAML file" and we can't do that here. (Probably a man page thing.)